src/components/pages/Offers/Offers/OfferItem/OfferItem.jsx

```diff
@@ -147,14 +149,6 @@ OfferItem.propTypes = {
   isSelected: PropTypes.bool,
   offer: PropTypes.shape().isRequired,
```

Comment:
détailler la shape en question c'est toujours bien quand même : 
```suggestion
  offer: PropTypes.shape({
      venue:  PropTypes.shape({
       ....
      }),
      stocks: PropType.arrayOf(PropTypes.shape({
       ....
      }))
  }).isRequired,
```